src/codeu/chat/common/BasicView.java

```diff
@@ -58,4 +59,14 @@
   //   Return all the interests objects.
   Collection<Interests> getInterests();
 
+  // GET USER ACCESS LEVELS
+  //
+  //   Returns all the interests
```

Comment:
I'd critique this comment (oh, the dangers of copying and pasting...), but as I commented above, I think the whole `getAccessLevels()` thing can be dropped.